CUSTOMERS_ANALYSIS/readme.md

```diff
@@ -0,0 +1,51 @@
+# Customer Analysis
+
+<ins>**Overview**</ins>
+
+The primary purpose of a business is to make a profit, which is crucial for its survival and growth.
+Data collection and analysis can enable a company understand its customer base and
+market trends, ultimately leading to more targeted marketing strategies,
+efficient resource allocation and enhanced profitability.
+
+<ins>**Data Structure**</ins>
+
+The data follows Medallion architecture, see definition by **[Databricks](https://www.databricks.com/glossary/medallion-architecture)**
+
+The bronze(raw) table consists of two tables(cvs file):customers and orders table
+
+**Table 1: Customers Information**
+
+- id(int):is a unique identifier for each customer,
+- firstname(text):stores the customer's given first name
+- lastname(text): stores the customer's given last name
+- email(text): holds the customer's email address
+- city(text): indicates the customer's city of residence
+- birth_date(text): records the customer's date of birth
+
+**Table 2: Orders Information**
+
+- orderID(text): serves as a unique identifier for each order placed
+- customer_id(int): links to the ID of the customer who placed the order
+- order_amount(int): the value of the order
+- currency(text):the currency in which the order was made
+- full_country_name(text): the country of the order's origin
+- purchase_date(varchar): indicates date of order
+- product_category(text):the category the order belongs to
+
+<ins>**Code Desription**</ins>
+
+The script starts with running codes for cleaning the bronze customer table and orders table into an silver table for the following analysis following analysis:
+
+1. Identification of the anual top spenders accross countries.This analysis helps in understanding which geographical regions are the most lucrative and identifying key market segments.
+2. International top spenders to understand the company's client base in the international market.This is essential for strategizing international market expansion.
+3. Lastly to get insights on customers lifetime purchase to get overall spending behaviour.This can be used for assessing long-term customer value, understanding customer loyalty trends, and developing strategies for customer retention and value maximization.
+
+<ins>**Findingds**</ins>
+
+1. The top 10 spenders generated about 3% profits of the entire client base
+2. The internaltional market showed the highest purchase from Asia
+3. The lifetime analysis gave a value of 19,000 for the highest purchser
+
+<ins>**Conclusion**</ins>
```

Comment:
The conclusion is too vague, I would either:
a) remove it
b) Add specific proposals based on the insights above (e.g. "We propose that a VIP customer service group be created to cater to top spenders, or something like this..)